app.R

```diff
@@ -269,7 +268,7 @@ ui <- navbarPage(title = "MetaDTA: Diagnostic Test Accuracy Meta-analysis",
                                            assessment data is not included in the file the covariates should be entered starting at the", tags$strong("seventh"), "column. If quality assessment
                                            data is included in the file the covariate data should be entered starting at the", tags$strong("fourteenth"), "column. Multiple covariates can be entered."),
                                          br(),
-                                         p("Note: Excel files should be saved in csv format and the separator option 'comma' selected for upload."),
+                                         p("Both Excel (.xlsx) and Comma Seperated Value (.csv) files are accepted."),
```

Comment:
This would be better moved to under the "Please select a file" button as it is hidden in all the text

Reply:
I have moved this to the file input help.